Dockerfile

```diff
@@ -4,6 +4,10 @@ FROM --platform=linux/amd64 synthetixio/docker-e2e:18.16-ubuntu as base
 RUN mkdir /app
 WORKDIR /app
 
+RUN apt update && apt install -y nginx
+
+COPY nginx.conf /etc/nginx/sites-available/default
+
```

Comment:
I'm not sure if this is good place to add this piece. The issue is that people use this `Dockerfile` as a reference to run their setup and nginx in most cases is redundant there and is very specific to our use case.

Nevertheless, sooner or later there may be more cases with specific things to our setup, so maybe it's not a huge issue at all. We can provide proper examples in `synpress-examples` repo.